ambry-router/src/main/java/com.github.ambry.router/GetBlobOperation.java

```diff
@@ -1145,8 +1157,10 @@ private void handleSimpleBlob(BlobData blobData, byte[] userMetadata, ByteBuffer
           long startTimeMs = System.currentTimeMillis();
           decryptCallbackResultInfo = new DecryptCallBackResultInfo();
           progressTracker.initializeDecryptionTracker();
+          decryptJobMetricsTracker = new CryptoJobMetricsTracker(routerMetrics.decryptJobMetrics);
```

Comment:
need to understand why these are created at multiple places - a little hard to follow

Reply:
New CryptoJobMetricsTracker will be created for every encrypt or decrypt job and hence. For Puts, encryption happens only at one place (one method), where as decryption happens at 3 places (data chunk, simple and metadata chunk) and hence.